src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -759,7 +759,7 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
                           capability.odp_cap.mem_page_fault)) {
         odp = UCT_IB_MLX5DV_ADDR_OF(query_hca_cap_out, out,
                 capability.odp_cap.memory_page_fault_scheme_cap);
-        scheme = "memory";
+        scheme = "remote-access";
```

Comment:
What is the meaning of the change?

Reply:
Before:
`mlx5_0: ODP transport is enabled` or `mlx5_0: ODP memory is enabled` 
These messages are confusing because there is no such thing as ODP transport type (there are RC/DC/UD transport types), and also we don't really enable ODP here (because it's enabled by setting UCP param of which memory types to use nonblock registration)
After:
`mlx5_0: remote-access ODP is supported` or `mlx5_0: full ODP is supported`

Reply:
Maybe ODPv1/ODPv2?

Reply:
we also enable full odp with odpv1 if qp is not created by devx

Reply:
yes, we don't have other options - full or disabled

Reply:
@Artemy-Mellanox fixed